.github/workflows/label.yml

```diff
@@ -0,0 +1,19 @@
+# This workflow will triage pull requests and apply a label based on the
+# paths that are modified in the pull request.
+#
+# To use this workflow, you will need to set up a .github/labeler.yml
+# file with configuration.  For more information, see:
+# https://github.com/actions/labeler/blob/master/README.md
+
+name: Labeler
+on: [pull_request]
+
+jobs:
+  label:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/labeler@v2
+      with:
+        repo-token: "${{ secrets.GITHUB_TOKEN }}"
```

Comment:
Is this secret something I need to configure?

Reply:
I don't believe you need to do anything no.

Reply:
okay perfect

Reply:
FWIW, from https://github.com/spf13/cobra/issues/952#issuecomment-532200393:

> Better integration of permissions. Since Actions is built in GitHub, **it is possible to use secret tokens without explicitly managing/setting them**.